docs/feature-store/transformations.md

```diff
@@ -189,4 +189,83 @@ quotes_set.graph.add_step("MyMap", "multi", after="filter", multiplier=3)
 ```
 
 This uses the `add_step` function of the graph to add a step called `multi` utilizing `MyMap` after the `filter` step 
-that was added previously. The class is initialized with a multiplier of 3.
\ No newline at end of file
+that was added previously. The class is initialized with a multiplier of 3.
+
+## Supporting multiple engines
+
+MLRun supports multiple processing engines for executing graphs. These engines differ in the way they invoke graph
+steps. When implementing custom transformations, the code has to support all engines which are expected to run it. 
```

Comment:
steps. When implementing custom transformations, the code has to support all engines that are expected to run it.